test/unit/argument_parser/detail/version_check_debug_test.cpp

```diff
@@ -0,0 +1,10 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#undef NDEBUG // test in debug mode
```

Comment:
Isn't this an error if NDEBUG isn't set before?

Reply:
it was the other way around. This is fine but in release mode I got an "NDEBUG defined multiple times error"